flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/catalog/CatalogViewITCase.scala

```diff
@@ -307,6 +307,91 @@ class CatalogViewITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     // now we only have permanent view T3
     assertEquals(permanentViewData.sorted, TestCollectionTableFactory.RESULT.sorted)
   }
+
+  @Test
+  def testShowCreateView(): Unit = {
+    val sourceData = List(toRow(1, "1000", 2))
+    TestCollectionTableFactory.initData(sourceData)
```

Comment:
We have new infrastructure for creating source/sink tables that don't require these big runtime implementation sources, see `TableFactoryHarness`. You could use that and create the table using `TableDescriptor` / `tableEnv.createTable`.

Reply:
Thanks for the comment. I updated it.